__tests__/topics.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest');
+const app = require('../app/app');
+
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+  test('GET /api/topics - success', async () => {
+    const response = await request(app).get('/api/topics');
+    expect(response.status).toBe(200);
+    expect(response.body.topics).toBeInstanceOf(Array);
+    response.body.topics.forEach(topic => {
+      expect(topic).toHaveProperty('slug');
+      expect(topic).toHaveProperty('description');
+    });
+  });
+
+  test('GET /api/invalid-endpoint - 404 error', async () => {
+    const response = await request(app).get('/api/invalid-endpoint');
+    expect(response.status).toBe(404);
+  });
```

Comment:
Great cover of invalid endpoints. As it's for dealing with all the invalid endpoints, so it's better be outside of the describe of ('/api/topics'). Also it'll be good to wrap all the endpoints into one big describe

Reply:
Yeah wasn't sure if putting the tests into separate files would be better for the PRs, I'll add that into the branch i'm working on now, thanks Scarlett